bom-2.289.x/pom.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-4.0/ms256048(v=vs.100)?redirectedfrom=MSDN#encoding-declaration

@basil docs says encoding case does not matter.

Is it something I should fix in the script or just let it be lower cased by powershell xml writer?

Reply:
Seems fine to leave this lowercase.